dangerfile.ts

```diff
@@ -0,0 +1,45 @@
+import { danger } from 'danger';
+import * as logger from '@markbind/core/src/utils/logger';
+
+interface Couplings {
+  [key: string]: string[];
+}
+
+const couplings: Couplings = {
+  'packages/core/src/html/CustomListIconProcessor.ts': [
+    'docs/userGuide/syntax/lists.md',
+    'packages/cli/test/functional/test_site/testList.md',
+    'packages/cli/test/functional/test_site/expected/testList.html',
```

Comment:
Hmmm tbh i feel like there isn't a need to add the .html and .page-vue-render.js files here. Because they will have to updated otherwise the tests will fail automatically.